requirements.txt

```diff
@@ -1,4 +1,4 @@
-pulpcore>=3.49.0,<3.55
+pulpcore>=3.49.0,<3.70.0
```

Comment:
That's a bit dangerous and I think we haven't yet officially decided to go with 3.70.
Given the bindings discussion, we might actually discuss a shorter window this time.